apps/api/jwt_auth/handlers.py

```diff
@@ -27,6 +28,8 @@
 
 
 def jwt_payload_handler(user, issuer):
+    # This creates a payload but at the time of this comment it is only used
```

Comment:
the only reason it's here is because the JWT lib wants it registered as a callback. The reason it wants to register it is because it has the facility to generate JWTs, we just have it turned off for now. I suspect we may eventually turn it on say, for example, if we want to support client-side JWT sign-in where the agent trade a user+pass for a token.